roles/opensrp/meta/main.yml

```diff
@@ -7,6 +7,13 @@ dependencies: # noqa 701
   swap_swappiness: "{{ opensrp_swap_swappiness }}"
   when: opensrp_install_swap
 
+- role: openjdk-11
```

Comment:
@githengi does this mean that deployments that run the old spring version will also run OpenJDK 11?

Reply:
Yes, older deployments were using `geerlingguy.java`. 

`openjdk-11` is wrapper role for onaio.tomcat which invokes `geerlingguy.java`.

@jasonrogena  Is there a convention for naming wrapper role?

Reply:
@githengi the naming should reflect what the role does. If `openjdk-11` does more than just install specifically OpenJDK 11, I'd recommend you change the naming.